.github/workflows/linux-checks.yaml

```diff
@@ -117,7 +117,7 @@ jobs:
         working-directory: ${{env.GOPATH}}/src/github.com/insolar/assured-ledger/ledger-core
       - name: Run functional tests multiple times with race detector as local cloud
         if: always()
-        run: make test_func_cloud_race
+        run: make test_func_cloud_race > functional_test_cloud.log
```

Comment:
please redirect logger to file, not console

Reply:
will be fixed later on